@AntoinePrv do you know what's going on in the installation step here:

```
2023-08-03T15:52:18.6189668Z -- The C compiler identification is GNU 12.3.0
2023-08-03T15:52:18.7310082Z -- The CXX compiler identification is GNU 12.3.0
2023-08-03T15:52:18.7519350Z -- Detecting C compiler ABI info
2023-08-03T15:52:18.8561270Z -- Detecting C compiler ABI info - done
2023-08-03T15:52:18.8812359Z -- Check for working C compiler: $PREFIX/bin/x86_64-conda-linux-gnu-cc - skipped
2023-08-03T15:52:18.8821830Z -- Detecting C compile features
2023-08-03T15:52:18.8833128Z -- Detecting C compile features - done
2023-08-03T15:52:18.8905589Z -- Detecting CXX compiler ABI info
2023-08-03T15:52:18.9930618Z -- Detecting CXX compiler ABI info - done
2023-08-03T15:52:19.0195041Z -- Check for working CXX compiler: $PREFIX/bin/x86_64-conda-linux-gnu-c++ - skipped
2023-08-03T15:52:19.0200098Z -- Detecting CXX compile features
2023-08-03T15:52:19.0210550Z -- Detecting CXX compile features - done
2023-08-03T15:52:19.0286697Z CMake Error at CMakeLists.txt:9 (find_package):
2023-08-03T15:52:19.0287328Z   By not providing "FindSCIP.cmake" in CMAKE_MODULE_PATH this project has
2023-08-03T15:52:19.0289015Z   asked CMake to find a package configuration file provided by "SCIP", but
2023-08-03T15:52:19.0289486Z   CMake did not find one.
2023-08-03T15:52:19.0289634Z 
2023-08-03T15:52:19.0289858Z   Could not find a package configuration file provided by "SCIP" with any of
2023-08-03T15:52:19.0290234Z   the following names:
2023-08-03T15:52:19.0290324Z 
2023-08-03T15:52:19.0290479Z     SCIPConfig.cmake
2023-08-03T15:52:19.0290750Z     scip-config.cmake
2023-08-03T15:52:19.0290839Z 
2023-08-03T15:52:19.0291029Z   Add the installation prefix of "SCIP" to CMAKE_PREFIX_PATH or set
2023-08-03T15:52:19.0291272Z   "SCIP_DIR" to a directory containing one of the above files.  If "SCIP"
2023-08-03T15:52:19.0291787Z   provides a separate development package or SDK, be sure it has been
2023-08-03T15:52:19.0291984Z   installed.
2023-08-03T15:52:19.0292067Z 
2023-08-03T15:52:19.0292136Z 
2023-08-03T15:52:19.0292373Z -- Configuring incomplete, errors occurred!
2023-08-03T15:52:19.6368467Z Tests failed for scip-8.0.4-h4a32fe0_0.conda - moving package to /home/conda/feedstock_root/build_artifacts/broken
```

not sure if some particular configuration / step is expected?

--- a/recipe/bld.bat
+++ b/recipe/bld.bat
@@ -16,7 +16,8 @@ cmake -G"NMake Makefiles" ^
       -D ZLIB=ON ^
       -D READLINE=OFF ^
       -D EXPRINT=cppad ^
-      -D SYM=bliss
+      -D SYM=bliss ^
+      %CMAKE_ARGS%
 if errorlevel 1 exit 1
 cmake --build scipoptsuite-build --parallel "%CPU_COUNT%"
 if errorlevel 1 exit 1

--- a/recipe/build.sh
+++ b/recipe/build.sh
@@ -31,6 +31,7 @@ cmake -B scipoptsuite-build -S "${SRC_DIR}/scipoptsuite" \
       -D READLINE=OFF \
       -D SYM=bliss \
       -D EXPRINT=cppad \
-      -D CLIQUER=ON
+      -D CLIQUER=ON \
+      -D CMAKE_INSTALL_LIBDIR=$PREFIX/lib
 cmake --build scipoptsuite-build --parallel ${CPU_COUNT}
 cmake --install scipoptsuite-build --prefix "${PREFIX}"

--- a/recipe/build.sh
+++ b/recipe/build.sh
@@ -32,6 +32,6 @@ cmake -B scipoptsuite-build -S "${SRC_DIR}/scipoptsuite" \
       -D SYM=bliss \
       -D EXPRINT=cppad \
       -D CLIQUER=ON \
-      -D CMAKE_INSTALL_LIBDIR=$PREFIX/lib
+      ${CMAKE_ARGS}
 cmake --build scipoptsuite-build --parallel ${CPU_COUNT}
 cmake --install scipoptsuite-build --prefix "${PREFIX}"

--- a/recipe/run_scip_test.bat
+++ b/recipe/run_scip_test.bat
@@ -1,4 +1,4 @@
-cmake -G"NMake Makefiles" -B build -S scipoptsuite/scip/examples/Queens -D CMAKE_BUILD_TYPE=Release
+cmake -G"NMake Makefiles" -B build -S scipoptsuite/scip/examples/Queens %CMAKE_ARGS%
 if errorlevel 1 exit 1
 
 cmake --build build --parallel "%CPU_COUNT%"

--- a/recipe/run_scip_test.sh
+++ b/recipe/run_scip_test.sh
@@ -4,7 +4,7 @@ set -o errexit
 set -o pipefail
 set -o nounset
 
-cmake -B build -S scipoptsuite/scip/examples/Queens -D CMAKE_BUILD_TYPE=Release
+cmake -B build -S scipoptsuite/scip/examples/Queens ${CMAKE_ARGS}
 cmake --build build --parallel ${CPU_COUNT}
 ./build/queens 5
 

